.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Bug report
```

Comment:
oh wait i think you might have the older version of the github templates? unless i'm using old ones lol, but at least the [github](https://github.com/fivetran/dbt_github/tree/main/.github) package has slightly different templates (and there's no question one)

Reply:
oh no good catch, i copied and pasted without doing a pull on a different repo! thanks for the fix